thetis/coupled_timeintegrator_2d.py

```diff
@@ -92,9 +89,7 @@ def _create_tracer_integrator(self):
                     semi_implicit=self.options.timestepper_options.use_semi_implicit_linearization,
                     theta=self.options.timestepper_options.implicitness_theta)
             else:
-                self.timesteppers.tracer = self.tracer_integrator(solver.eq_tracer, solver.fields.tracer_2d, fields, solver.dt,
-                                                                  bnd_conditions=solver.bnd_functions['tracer'],
-                                                                  solver_parameters=self.options.timestepper_options.solver_parameters_tracer,)
+                self.timesteppers.tracer = self.tracer_integrator(solver.eq_tracer, solver.fields.tracer_2d, fields, solver.dt, bnd_conditions=solver.bnd_functions['tracer'], solver_parameters=self.options.timestepper_options.solver_parameters_tracer,)
```

Comment:
also here the old split is more legible

Reply:
I have changed this and committed it to the branch